docs/UserGuide.md

```diff
@@ -164,96 +136,101 @@ Item | Description
+  * e.g. `find property Condo` can also be entered as `find properties Condo`.
 
-Under the [`find` command](#locating-propertiesbuyers-by-name-tags-price-find), while each individual argument is optional, it is not valid to leave out all arguments. For example, `find properties` alone is invalid. See the [`find` command](#locating-propertiesbuyers-by-name-tags-price-find) for more detailed examples.
+* Words in `UPPER_CASE` must be replaced with the corresponding field by the user.<br>
+  * e.g. in `add property n/PROPERTY_NAME`, `PROPERTY_NAME` is a parameter that can be used as `add property n/Beautiful Condo`.
 
-</div>
+* `[]` : Items in square brackets are optional.<br>
+  * e.g `n/PROPERTY_NAME [t/TAG]` can be used as `n/Beautiful Condo t/condo` or as `n/Beautiful Condo`.
 
 * `…` : Items with `…` after them can be repeated any number of times.<br>
   * e.g. `[t/TAG]…` can be omitted, used once:`t/condo`, twice:`t/condo t/family` or more times.
 
-* `( | )` : Items in circle brackets `()` separated by `|` means that you can only choose 1 of the partitioned items <br>
-  * e.g. `([t/TAG] | [ta/TAG_TO_ADD] [t/TAG_TO_DELETE])` can be used as `t/condo`, `ta/condo td/small condo` but not `t/condo ta/condo`, `t/condo td/small condo`, `t/condo ta/condo td/small condo`
-
-
-* Parameters can be in any order.<br>
-  * e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* Our commands use slashes `/` as prefix identifiers, so do not use `/` in command arguments.
-  * e.g. `edit property 3 s/Anish s/o Reyaz` will result in an error message displayed in the output box.
+* `( | )` : Items in round brackets `()` separated by `|` means that you can only choose 1 of the items <br>
+  * e.g. `([t/TAG] | [ta/TAG_TO_ADD] [t/TAG_TO_DELETE])` can be used as `t/condo` or `ta/condo td/small condo` but not `t/condo ta/condo` or `t/condo td/small condo`
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   * e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
 ------------------
 
 ## Valid properties/buyers
+This section describes the requirements of certain fields.
 
-Here are some fields that are shared amongst commands.
-
-### Property/Buyer names
-* Names must start with a letter or number, and only contain alphanumerical characters, spaces and hyphens (`-`).
+### Property/buyer names
+* Names must only contain alphanumerical characters, spaces and hyphens (`-`).
 * Names have a maximum allowed length of 50.
 * Buyers with identical names are not allowed.
 
 ### Property addresses
-* Addresses must start with a letter or number, and only contain alphanumerical characters, and the following special characters:
-  * spaces,
-  * hyphens (`-`),
-  * commas (`,`),
-  * semicolons (`;`),
-  * and hashes (`#`).
+* Addresses must only contain alphanumerical characters or the following special characters:
```

Comment:
Same here, can add a line about blank input being invalid

Reply:
I think no need to be exhaustive